### ISSUE TITLE
Unable to add an MDX Tuple to an axis directly

### ISSUE THREAD

**Issue by rclapp** [cubewise-code/mdxpy#13]
Trying to add an MDX tuple to an axis directly results in an exception or invalid MDX

1) Add using HierarchySet methods
`
mdx_tuple = mdxpy.MDXTuple.of(mdxpy.Member.of('Year','2019'), mdxpy.Member.of('Period','FY Total'))
query.columns_non_empty().add_hierarchy_set_to_column_axis(mdx_tuple)
`

to_mdx: 
SELECT
NON EMPTY ([YEAR].[YEAR].[2019],[PERIOD].[PERIOD].[FYTOTAL]) ON 0,
*Note the missing {} around the tuple

2) Add using MDX Tuple Methods

`mdx_tuple = mdxpy.MDXTuple.of(mdxpy.Member.of('Year','2019'), mdxpy.Member.of('Period','FY Total'))
query.columns_non_empty().add_member_tuple_to_columns(mdx_tuple)
`
AttributeError: 'MdxTuple' object has no attribute 'unique_name'

I think we need a new method to add a tuple to an axis directly.

**Comment by MariusWirtz** [comment]
Hi @rclapp,

I see where you are coming from. When you have an instance of `MdxTuple` you can't consume this in the MdxBuilder.
I agree that's a design flaw. 

You can kinda work around it, by "unpacking" the `members` property of the `mdx_tuple` and pass it as the `*args` to the `add_member_tuple_to_columns` function, but that's not an ideal solution. I also think there should be an easier way. 

Please feel free to open a MR for this one. 

``` python
from mdxpy import MdxBuilder, MdxTuple, Member

mdx_tuple = MdxTuple.of(
    Member.of('Year', '2019'),
    Member.of('Period', 'FY Total'))

query = MdxBuilder.from_cube("Cube") \
    .columns_non_empty() \
    .add_member_tuple_to_columns(*mdx_tuple.members)

print(query.to_mdx())

```

**Comment by MariusWirtz** [comment]
Fixed with #20
